db/schema.rb

```diff
@@ -180,7 +180,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["name", "resource_type", "resource_id"], name: "index_roles_on_name_and_resource_type_and_resource_id"
-    t.index ["resource_type", "resource_id"], name: "index_roles_on_resource_type_and_resource_id"
+    t.index ["resource_type", "resource_id"], name: "index_roles_on_resource"
```

Comment:
why did this index name change?

Reply:
Just cleared out my db and rerun migrations; this is how it generated.

Reply:
weird.